org/istio.yaml

```diff
@@ -560,7 +559,6 @@ orgs:
           WG - Docs Maintainers:
             description: Maintainers of the Docs working group.
             maintainers:
-            - geeknoid
```

Comment:
mtail can you add `frankbu` here otherwise there is no maintainer.

Reply:
maintainer here means they are a GitHub admin - it's. pretty confusing because it's. mapping the GitHub API directly, as might want to add another API on top of those.